rust-executor/src/holochain_service/mod.rs

```diff
@@ -307,23 +307,23 @@ impl HolochainService {
             let config = ConductorConfig::load_yaml(&conductor_yaml_path)?;
             config
         } else {
-            let mut config = ConductorConfig::default();
+        let mut config = ConductorConfig::default();
             config.environment_path = PathBuf::from(local_config.conductor_path.clone()).into();
             config.admin_interfaces = None;
 
             let mut kitsune_config = KitsuneP2pConfig::default();
-            // let mut tuning_params = KitsuneP2pTuningParams::default();
+            let mut tuning_params = KitsuneP2pTuningParams::default().as_ref().clone();
```

Comment:
Looks great @fayeed thanks! 

Lets leave the params here as default and just have our custom params commented out for now. Also need to see why the languages are not compiling in CI.